activerecord/lib/active_record/fixture_set/model_metadata.rb

```diff
@@ -21,8 +21,7 @@ def has_primary_key_column?
       end
 
       def timestamp_column_names
-        @timestamp_column_names ||=
-          %w(created_at created_on updated_at updated_on) & @model_class.column_names
+        @timestamp_column_names ||= @model_class.all_timestamp_attributes_in_model
```

Comment:
Looks like `all_timestamp_attributes_in_model` already caches the columns.

Reply:
Good catch! I'll remove the extra cache.